src/settings/pch.h

```diff
@@ -15,3 +15,4 @@
 #pragma pop_macro("GetCurrentTime")
 #include <strsafe.h>
 #include <Shlwapi.h>
+#include <common/Telemetry/ProjectTelemetry.h>
```

Comment:
The way telemetry is enabled, requires that the `ProjectTelemetry.h` is included like this:
```
#include <ProjectTelemetry.h>
```
because in the release pipeline, we add another file, that contains the actual real telemetry settings and it's loaded before the one in `common`